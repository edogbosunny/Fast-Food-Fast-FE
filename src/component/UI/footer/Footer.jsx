import React from 'react';

export default function Footer() {
  return (
    <footer className="footer">
      <p className='ash'>
An Andela project by Edogbo Sunny | &copy;
        {new Date().getFullYear()}
      </p>
    </footer>
  );
}
