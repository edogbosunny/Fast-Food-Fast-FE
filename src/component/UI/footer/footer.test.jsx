import { shallow } from 'enzyme';
import React from 'react';
import Footer from './Footer';

describe('Footer Component', () => {
  const wrapper = shallow(
    <Footer />,
  );

  it('should shallow the footer component!', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
