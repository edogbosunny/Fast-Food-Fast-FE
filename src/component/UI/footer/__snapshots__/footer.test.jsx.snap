// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should shallow the footer component! 1`] = `
<footer
  className="footer"
>
  <p
    className="ash"
  >
    An Andela project by Edogbo Sunny | Â©
    2019
  </p>
</footer>
`;
